DTDL/ssm_stream_deviceinterfaceprotocol.json

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:stream:deviceinterfaceprotocol;1",
+  "@type": "Interface",
+  "extends": "dtmi:io:tributech:stream:base;1",
+  "displayName": "Device Interface Protocol Stream",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "MinInterval",
+      "schema": "duration",
+      "writable": true,
+      "displayName": "Minimum Time Interval",
+      "description": "Specifies the minimal time-interval of values which can be provided for this stream.",
```

Comment:
what unit is the interval

Reply:
The schema is `duration`. A duration in specified to be ISO 8601 format. More details can be found [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md).
Therefore I think we don't need an explicit `Unit`.